forge-gui/res/cardsfolder/e/ersta_friend_to_all.txt

```diff
@@ -1,13 +1,13 @@
 Name:Ersta, Friend to All
 ManaCost:W U B R G
-Types:Legendary Planeswalker Estra
+Types:Legendary Planeswalker Ersta
```

Comment:
Filename wrong too

Reply:
I believe I already corrected that: 
![image](https://github.com/user-attachments/assets/53987133-d47f-4d15-b915-55903af48527)